packages/client/modules/email/components/MeetingSummaryEmailRootSSR.tsx

```diff
@@ -34,36 +37,55 @@ export const meetingSummaryUrlParams = {
 
 const MeetingSummaryEmailRootSSR = (props: Props) => {
   const {appOrigin, environment, meetingId} = props
+  const queryRef = useQueryLoaderNow<MeetingSummaryEmailRootSSRQuery>(
```

Comment:
-1 i believe we'll have to change this because now we're importing more than just a type, we're importing an object.

This is problematic because to build `meetingSummaryEmailRootSsrQuery`, relay-compiler needs a graphql schema & to build a graphql schema, graphql must traverse the /graphql/public dir, which eventually depends on this file. This creates a circular dependency.

I'd suggest using a lazy-load pattern. See https://github.com/ParabolInc/parabol/blob/0cb19afc1043501a55a990e3a838c1792840e1bb/packages/client/components/DemoMeetingRoot.tsx#L30

Reply:
make sense! I updated to use `useLazyLoadQuery` to load data directly and avoid circular dependency